src/query/procedure/module.cpp

```diff
@@ -12,6 +12,7 @@
 #include "query/procedure/module.hpp"
 
 #include <filesystem>
+#include <fstream>
```

Comment:
Do we need this import?

Reply:
Turns out it was needed, one of the refactors deprived fstream from module.cpp. This fixes the build.